config/locales/fr.yml

```
@@ -0,0 +1,48 @@
+---
+fr:
+  decidim:
+    admin:
+      filters:
+        initiatives:
+          state_eq:
+            values:
+              illegal: Illégale
+              invalidated: Invalide
+    transparent_trash:
+      initiatives:
+        filters_small_view:
+          close_modal: Fermer la fenêtre
+          filter_by: Rechercher
+          unfold: Voir la suite
+          filter: Filtrer
+        index_header:
+          go_back: Retour à la liste des pétitions
+        filters:
+          search: Rechercher
+    initiatives:
+      initiatives:
+        show:
+          illegal:
+            description: Contenu de la description modéré
+            title: Contenu du titre modéré
+      admin_log:
+        initiative:
+          invalidate: "%{user_name} published the %{resource_name} initiative as 'Invalidated'"
+          illegal: "%{user_name} published the %{resource_name} initiative as 'Illegal'"
```

These lines are not translated
```suggestion
          invalidate: "%{user_name} a publié la pétition %{resource_name} en tant que 'Invalide'"
          illegal: "%{user_name} a publié la pétition %{resource_name} en tant que 'Illégale'"
```